config/metrics_endpoints.json

```diff
@@ -1,635 +1,1111 @@
+        "coverall": "",
+        "github": [
+            "ec.europa.eu"
+        ],
+        "docker": [
+            "fiware/domibus-tomcat"
+        ]
+    },
+    {
+        "enabler": "Draco",
+        "catalogue": "",
+        "academy": "",
+        "readthedocs": "fiware-draco",
+        "helpdesk": "Draco",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "ging/fiware-draco"
+        ],
+        "docker": [
+            "ging/fiware-draco"
+        ]
+    },
+    {
+        "enabler": "FIWARE's Extended CKAN",
+        "catalogue": "fiware-ckan-extensions",
+        "academy": "CKAN",
+        "readthedocs": "fiware-ckan-extensions",
+        "helpdesk": "CKAN",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "FIWARE-TMForum/ckanext-baepublisher",
+            "conwetlab/FIWARE-CKAN-Extensions",
+            "conwetlab/ckanext-datarequests",
+            "conwetlab/ckanext-oauth2",
+            "conwetlab/ckanext-privatedatasets",
+            "conwetlab/ckanext-right_time_context",
+            "conwetlab/ckanext-wirecloud_view"
+        ],
+        "docker": [
+            "fiware/ckan-extended"
+        ]
+    },
+    {
+        "enabler": "Fast-RTPS",
+        "catalogue": "fast-rtps",
+        "academy": "",
+        "readthedocs": "eprosima-fast-rtps",
+        "helpdesk": "FastRTPS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "eProsima/Fast-RTPS"
+        ],
+        "docker": []
+    },
+    {
+        "enabler": "FiVES - Flexible Virtual Environment Server",
+        "catalogue": "synchronization-fives",
+        "academy": "FiVES",
+        "readthedocs": "fives",
+        "helpdesk": "Fives",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "fives-team/FiVES"
+        ],
+        "docker": [
+            "tospie/fives"
+        ]
+    },
+    {
+        "enabler": "FogFlow",
+        "catalogue": "fogflow",
+        "academy": "",
+        "readthedocs": "fogflow",
+        "helpdesk": "FogFlow",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "smartfog/fogflow"
+        ],
+        "docker": [
+            "fogflow/designer",
+            "fogflow/worker",
+            "fogflow/master",
+            "fogflow/broker",
+            "fogflow/discovery",
+            "fogflow/sum",
+            "fogflow/facecounter",
+            "fogflow/facefinder",
+            "fogflow/recommender",
+            "fogflow/connectedcar",
+            "fogflow/privatesite",
+            "fogflow/publicsite",
+            "fogflow/anomaly",
+            "fogflow/counter",
+            "fogflow/dummy",
+            "fogflow/nodejs"
+        ]
+    },
+    {
+        "enabler": "GIS Data Provider ",
```

Comment:
"enabler": "GIS Data Provider"

Reply:
Fixed in https://github.com/flopezag/fiware-tsc-dashboard/pull/38/commits/55ea9ac2cb5f6bb623a5b2897a9d1e5dede5c585